big_tests/tests/service_domain_db_SUITE.erl

```diff
@@ -1179,3 +1267,7 @@ assert_rest_db_error({Result, Msg}) ->
 
 dummy_auth_host_type() ->
     <<"dummy auth">>. %% specified in the TOML config file
+
+random_domain_name() ->
+    Suffix = list_to_binary(integer_to_list(rand:uniform(1000000000))),
```

Comment:
First thing is that `integer_to_binary/1` exist, and second, maybe simply use `erlang:unique_integer/1`, that one is ensured to be unique for the whole node already, and it's way faster to calculate than operations in the `rand` module.
```suggestion
    Suffix = integer_to_binary(rand:uniform(1000000000)),
```